src/web/middleware/sentry.rs

```diff
@@ -52,6 +55,28 @@ pub struct SentryWrapperMiddleware<S> {
     service: Rc<RefCell<S>>,
 }
 
+impl<S, B> SentryWrapperMiddleware<S>
+where
+    S: Service<Request = ServiceRequest, Response = ServiceResponse<B>, Error = Error> + 'static,
+    S::Future: 'static,
+    B: 'static,
+{
+    fn clean_uri(&self, uri: &Uri) -> String {
+        // Remove the variable userID and batch tokens from the URI tags
+        lazy_static! {
+            // most values are either "=true" or a lower case hex string.
+            static ref RE: regex::Regex = regex::Regex::new(r"=[a-z\d]{5,}").unwrap();
+        }
+        let trimmed = uri
+            .to_string()
+            .split('/')
```

Comment:
I think these have to be "

Reply:
They do, in spite of clippy saying otherwise. I've added a bit of a crap patch with comment to deal with that.